packages/grafana-ui/src/components/Button/ButtonContent.tsx

```diff
@@ -1,20 +1,30 @@
 import React from 'react';
 import cx from 'classnames';
+import { css } from 'emotion';
+import { stylesFactory } from '../../themes';
+
+const getStyles = stylesFactory(() => ({
+  content: css`
+    white-space: nowrap;
```

Comment:
Prevents button text to be wrapped which would mean the text overflows the button.